app/controllers/visualizations_controller.rb

```diff
@@ -3,20 +3,47 @@ class VisualizationsController < ApplicationController
   include HeatmapHelper
 
   def visualization
-    # TODO: (gdingle): dispatch by type
-    # type = params[:type]
-
+    # TODO: (gdingle): refactor off of heatmap
     @heatmap_data = heatmap
 
     id = params[:id]
     if id
       vis = Visualization.find(id)
+
+      # TODO: (gdingle): does this makes sense for non-heatmap?
```

Comment:
Yes, we plan to have more visualizations with multiple samples (e.g. world map).
Phylo tree visualizations (possibly with some refactoring) will also need customization and be related to multiple samples.

Reply:
👍